develop-docs/README.md

```diff
@@ -128,3 +128,14 @@ iOS 12 opens a risk of introducing bugs, which has already happened in the past,
 the iOS 12 simulator a try.
 
 Related to [GH-2218](https://github.com/getsentry/sentry-cocoa/issues/2218)
+
+### Adding Swift code in the project
+
+Date: October 1st 2022
+Contributors: @brustolin
+
+A Sentry SDK started to be [written in Swift once,](https://github.com/getsentry/raven-swift) but due to ABI not being stable at that time, it got dropped. Since then Swift 5.1 landed and we got ABI stability. We’ve considered adding Swift to our sentry.cocoa SDK since then, but because of some of the trade offs, we’ve postponed that decision.
```

Comment:
`l`: Swift 5.0 brought ABI stability, see https://www.swift.org/blog/swift-5-released/
```suggestion
A Sentry SDK started to be [written in Swift once,](https://github.com/getsentry/raven-swift) but due to ABI not being stable at that time, it got dropped. Since then Swift 5.0 landed and we got ABI stability. We’ve considered adding Swift to our sentry.cocoa SDK since then, but because of some of the trade offs, we’ve postponed that decision.
```